sdks/python/apache_beam/dataframe/pandas_top_level_functions.py

```diff
@@ -38,8 +38,14 @@ def wrapper(target, *args, **kwargs):
 
 
 def _defer_to_pandas(name):
+  func = getattr(pd, name)
+
   def wrapper(*args, **kwargs):
-    res = getattr(pd, name)(*args, **kwargs)
+    if any(isinstance(arg, frame_base.DeferredBase)
+           for arg in args + tuple(kwargs.values())):
+      return frame_base._elementwise_function(func, name)(*args, **kwargs)
```

Comment:
Is this always safe?

Reply:
I guess there's a concern that a top-level function might not operate element-wise on pandas args. I could make this a separate function, e.g. `_defer_to_pandas_elementwise` (and while I'm at it look over the other `_defer_to_pandas` usages and see if they should be switched).

Reply:
Done. I took a look at the existing `_defer_to_pandas` usages and none of them accept pandas objects. I did find `to_numeric`, which can use this.

I also added a commit to run to_numeric and to_datetime doctests.

Reply:
Although on second thought maybe we shouldn't support to_numeric since its proxy is data-dependent..

Reply:
Removed to_numeric, and I'll merge this.